auditsweeper/auditsweeper.py

```
@@ -0,0 +1,87 @@
+    required_vars = [
+        "db-host", "db-port", "db-su-user",
+        "postgres-password", "log-age-days"
+    ]
+
+    env_vars = {var: os.getenv(var) for var in required_vars}
+
+    # Check if all environment variables are set
+    if all(env_vars.values()):
+        print("Using credentials from environment variables.")
+        return env_vars
+    else:
```

Ruff (0.16.3)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)